# Array to store free parameters in
ParameterInfo[] params
